Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -257,10 +258,10 @@ namespace Fuse.Internal
 			}
 
 			public void Reset()
-			{
-				//it's easier not to support this since it would involve tracking the initial iterator state (added overhead)
-				//or it would incorrectly iterate the current source, not the one at the time of creation
-				throw new Exception( "Reset not supported" );
+			{	
+				_first = true;
+				if (_mode == MiniListMode.List)
+					_iter.Reset();
 			}
```

Comment:
No, this is just a somewhat unrelated change to fix a few test cases that somehow managed to use Reset. It wasn't too hard to give Reset a reasonable behaviour, so I did.